suse/publish/gen.rb

```diff
@@ -24,6 +24,10 @@
 </pre>
 
 <p>
+You will need to explicitly install a Java runtime environment, because Oracle's Java RPMs are incorrect and fail to register as providing a java dependency. 
```

Comment:
Could this list the minimum JRE by Jenkins version, or should we just say "Java 8 or GTFO"?

FTR:

* 2.54 (2017-04) and newer: Java 8
* 1.612 (2015-05) and newer: Java 7
* 1.520 (2013-06) and newer: Java 6

So let's plan the Java 9 dependency for March 2019 :-)